my-app/src/App.tsx

```diff
@@ -0,0 +1,52 @@
+import './App.css';
+
+const USERS: User[] = [
+  { name: "thanh", position: "$1", start_date: '12 Nov 2024' },
+  { name: "kim", position: "$1", start_date: '12 Nov 2024' },
+  { name: "nguyen", position: "$1", start_date: '12 Nov 2024' },
+  { name: "anh", position: "$1", start_date: '12 Nov 2024' }
+];
+
+interface User {
+  name: string;
+  position: string;
+  start_date: string;
+}
+
+const UserRow = ({ user }: { user: User }) => {
+  const name = user.name;
```

Comment:
De-structuring before use:
```ts
const { name, position, startDate } = user;
```

btw, should be `startDate` instead of `start_date`. Recheck naming convention.

Reply:
fixed